app/controllers/admin/dashboards_controller.rb

```diff
@@ -29,7 +37,8 @@ def create
     end
 
     def destroy
-
+      require 'pry' ; binding.pry
```

Comment:
Remove debugger entry point `binding.pry`.<br>Do not use semicolons to terminate expressions.<br>Space found before semicolon.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.